tests/test_project.py

```diff
@@ -146,6 +146,41 @@ def mock_project(self):
         return project
 
 
+@unittest.skipIf(six.PY2, 'Only check performance on Python 3')
+class ProjectStatusTest(BaseProjectTest):
```

Comment:
Are there ay other tests that we could potentially move into this test case? Otherwise I'd suggest to rename to `ProjectStatusPerformanceTest`.

Reply:
Yes I had that originally and accidentally changed that. I'll switch it back.